src/commands/start_project.rs

```diff
@@ -0,0 +1,56 @@
+use std::fs;
+use std::env;
+use std::fs::File;
+use std::io::prelude::*;
+
+async fn template_contents(project_name: &str) -> (String,String){
+	let ftd = format!("-- import: fpm\n\n-- fpm.package: {}", project_name);
+	let index = "-- ftd.text: Hello world".to_string();
+
+	(ftd,index)
+}
+
+async fn write_file(file_name: &str, dir: &str, content: &str){
+	let file_path = format!("{}/{}",dir,file_name);
+	let mut fp = File::create(file_path)
+		.expect("Could not create file!");
+
+	fp.write_all(content.as_bytes())
+		.expect("Could not write contents into the file!");
+}
```

Comment:
This function declares itself as `async` but is not actually async. Use `tokio::fs` for doing async file system operations.